source/Calamari.Shared/Integration/Proxies/CommandLineToolsProxyEnvironmentVariables.cs

```diff
@@ -0,0 +1,80 @@
+using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Net.Sockets;
+
+namespace Calamari.Integration.Proxies
+{
+    public class CommandLineToolsProxyEnvironmentVariables
```

Comment:
This makes sense since you need the proxy configuration in the .NET Runtime/process.